index.js

```diff
@@ -10,6 +10,7 @@ function generate(argIndex) {
 		var promise = thunk(value);
 		thunk.then = promise.then.bind(promise);
 		thunk.catch = promise.catch.bind(promise);
+		thunk._actualPromise = promise;
```

Comment:
Node.js is not following any spec for the unhandled rejection thing though. They even have different event names than the official spec used in the browser. Maybe open an issue on Node.js about it?